.github/workflows/test-application.yaml

```diff
@@ -28,3 +33,121 @@ jobs:
               run: curl "https://cs.symfony.com/download/php-cs-fixer-v2.phar" --output ${{ steps.php-cs-fixer-path.outputs.path }} && chmod +x ${{ steps.php-cs-fixer-path.outputs.path }}
             - name: Run php-cs-fixer
               run: ./${{ steps.php-cs-fixer-path.outputs.path }} fix --dry-run --diff
+
+    test:
+        name: "Run tests with php ${{ matrix.php-version }} (${{ matrix.phpcr-transport }})"
+        runs-on: ubuntu-latest
+
+        env:
+            JACKRABBIT_VERSION: 2.12.0
+            SYMFONY_DEPRECATIONS_HELPER: weak
+            SYMFONY__DATABASE__HOST: 127.0.0.1
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '5.5'
+                    - '7.0'
+                database: [ mysql ]
+                phpcr-transport: [ doctrinedbal ]
+                composer-flags: [ '' ]
+                test-flags: [ '' ]
+                php-extensions: [ 'ctype, iconv, mysql, gd' ]
+                tools: [ 'composer:v1' ]
+                env:
+                    - {
+                          SYMFONY__PHPCR__TRANSPORT: doctrinedbal,
+                          SYMFONY__DATABASE__DRIVER: pdo_mysql,
+                          SYMFONY__DATABASE__PORT: 3306,
+                    }
+                include:
```

Comment:
not sure if possible, but i feel like it would be more explicit and predictable if we define all "environments" in the `include`

Reply:
I don't know ... will try that